CFC_WebApp/clients/leaderboard/result_template.html

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
+
+<style>
+
+  .alignleft {
+    float: left;
+    font-weight: bold;
+  }
+
+  .alignright {
+    float: right;
+  }
+=
+
+</style>
+
+<html ng-app="e-mission-leaderboard">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+    <title>Leaderboard</title>
+    <script src="clients/leaderboard/front/d3.js" charset="utf-8"></script>
+    <script src="clients/leaderboard/front/bullet.js"></script>
+    <script src="clients/choice/front/ionic.bundle.js" charset="utf-8"></script>
+    <script src="clients/leaderboard/front/client.leaderboard.display.js" charset="utf-8"></script>
+
+    <link rel="stylesheet" href="clients/choice/front/ionic.min.css">
+  </head>
+  <body ng-controller="RootCtrl" >
+    <div class="list">
+        <body onLoad='showscores()'></body>
+    </div>
+    <script>
+        var currScore = {{currScore}};
```

Comment:
Now that this has been pulled out into the javascript file, can the showscores() function be removed from here? It is OK if you don't know the answer - just file an issue to clean it up later so that we don't forget.